pages/en/supported-networks/arweave.mdx

```diff
@@ -0,0 +1,199 @@
+
+## AssemblyScript Mappings
+
+The handlers for processing events are written in [AssemblyScript](https://www.assemblyscript.org/).
+
+Arweave indexing introduces Arweave-specific data types to the [AssemblyScript API](https://thegraph.com/docs/en/developer/assemblyscript-api/).
+
+```tsx
+class ProofOfAccess {
+    option: string
+    txPath: Bytes
+    ataPath: Bytes
+    chunk: Bytes
+}
+  
+class Block {
+     timestamp: u64
+     lastRetarget: u64
+     height: u64
+     indepHash: Bytes
+     nonce: Bytes
+     previousBlock: Bytes
+     diff: Bytes
+     hash: Bytes
+     txRoot: Bytes
+     txs: Bytes[]
+     walletList: Bytes
+     rewardAddr: Bytes
+     tags: Tag[]
+     rewardPool: Bytes
+     weaveSize: Bytes
+     blockSize: Bytes
+     cumulativeDiff: Bytes
+     hashListMerkle: Bytes
+     poa: ProofOfAccess
+}
+
+class Transaction {
+     format: u32
+     id: Bytes
+     lastTx: Bytes
+     owner: Bytes
+     tags: Tag[]
+     target: Bytes
+     quantity: Bytes
+     data: Bytes
+     dataSize: Bytes
+     dataRoot: Bytes
+     signature: Bytes
+     reward: Bytes
+}
+```
+
+Writing the mappings of an Arweave Subgraph is very similar to writing the mappings of an Ethereum Subgraph. For more information, click [here](https://thegraph.com/docs/en/developer/create-subgraph-hosted/#writing-mappings).
+
+## Deploying an Arweave Subgraph
+
+Once your subgraph has been created, you can deploy your subgraph by using the `graph deploy` CLI command after running the `graph create` CLI command:
+
+**Hosted Service**
+```bash
+graph create subgraph-name --product hosted-service # creates a subgraph on a local Graph Node (on the Hosted Service, this is done via the UI)
+```
+```bash
+graph deploy --node https://api.thegraph.com/deploy/ --ipfs https://api.thegraph.com/ipfs/ --access-token <your-access-token>
+```
+
+**Local Graph Node (based on default configuration):**
+```bash
+graph create subgraph-name --node http://localhost:8020
+```
+```bash
+graph deploy --node http://localhost:8020/ --ipfs http://localhost:5001
+```
+
+Once your subgraph has been deployed, it will be indexed by Graph Node. You can check its progress by querying the subgraph itself:
+```
+{
+  _meta {
+    block { number }
+  }
+}
+``` 
+
+## Querying an Arweave Subgraph
+
+The GraphQL endpoint for Arweave subgraphs is determined by the schema definition, with the existing API interface. Please visit the [GraphQL API documentation](https://thegraph.com/docs/en/developer/graphql-api/) for more information.
+
+## Example Subgraphs
+
+Here is an example subgraph for reference:
+- [Example subgraph for Arweave](https://github.com/graphprotocol/example-subgraph/tree/arweave-blocks-transactions)
```

Comment:
The example link https://github.com/graphprotocol/example-subgraph/blob/arweave-blocks-transactions/subgraph.yaml#L7 doesn't point to a valid Arweave example, todo?

Reply:
I have opened a PR. It is not merged yet: https://github.com/graphprotocol/example-subgraph/pull/51

Reply:
I merged that PR, so the link works now